userspace/libsinsp/user_event.h

```diff
@@ -24,6 +24,54 @@ along with sysdig.  If not, see <http://www.gnu.org/licenses/>.
 #include <string>
 #include <set>
 #include <unordered_map>
+#include <regex.h>
+
+//
+// scope utilities
+//
+class event_scope
+{
+public:
+	typedef std::vector<std::string> string_list_t;
+
+	static const std::string SCOPE_OP_AND;
+	static const string_list_t RESERVED_STRINGS;
+	static const string_list_t REPLACEMENT_STRINGS;
+
+	event_scope(const std::string& key = "", const std::string& value = "");
+
+	bool add(const std::string& key, const std::string& value, const std::string& op = SCOPE_OP_AND);
+
+	std::string get();
+
+	void clear();
+
+	// utility function to check that scope entry is valid;
```

Comment:
This comment is out of date, please update it. Also, the name is vague. How about check_key_format()?